.github/workflows/io.github.thinkle.Gourmet.yml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.thinkle.Gourmet
```

Comment:
This file should be at the root of the project: while we use it for the CI runner, it's fundamentally part of the Github infrastructure, and can be used elsewhere :)